schema/segmenter/test_schema.json

```diff
@@ -0,0 +1,110 @@
+{"$schema": "https://json-schema.org/draft/2020-12/schema",
+ "$id": "https://github.com/unicode/conformance/segmenter_test_schema.json'",
+ "title": "ICU Conformance test data description",
+ "description": "This documents the format of segmenter test data for execution",
+ "type": "object",
+ "properties": {
+   "additionalProperties": false,
+   "test_type": {
+     "description": "The name of the test",
+     "const": "segmenter"
+   },
+   "Test scenario": {
+     "type": "string",
+     "description":" Obsolete tag to be removed and replaced with test Type"
+   },
+   "source_file": {
+     "description": "origin of conformance data",
+     "type": "string"
+   },
+   "source_version": {
+     "description": "version of conformance data",
+     "type": "string"
+   },
+   "test_ environmment": {
+     "test_language":  {
+       "description": "Programming language of execution",
+       "type": "string"
+     },
+     "executor": {
+       "description": "path to the executor for this output",
+       "type": "string"
+     },
+     "test_type": {
+       "description": "the kind of tests run",
+       "type": "string"
+     },
+     "date_time": {
+       "description": "timestamp of output",
+       "type": "string"
+     },
+     "input_file": {
+       "description": "path to test data",
+       "type": "string"
+     },
+     "icu_version": {
+       "description": "ICU data version used in testing",
+       "type": "string"
+     },
+     "cldr_version": {
+       "description": "CLDR version used in testing",
+       "type": "string"
+     },
+     "test_count": {
+       "description": "Number of tests run",
+       "type": "integer"
+     }
+   },
+   "error_info": {
+     "description": "count of detected errors",
+     "type": "integer"
+   },
+   "platform": {
+     "description": "the executor's platorm",
+     "type": "integer"
+   },
+   "tests": {
+     "description": "list of N tests for segmenter",
+     "type": "array",
+     "items": {
+       "additionalProperties": false,
+       "type": "object",
+       "properties": {
+         "label": {
+           "description": "A numeric ID, unique for the set of tests",
+           "type": "string"
+         },
+         "input": {
+           "description": "Some text to be segmented",
+           "type": "string"
+         },
+          "hexhash": {
+            "description": "A hexadecimal hash code for the test without the label",
+            "type": "string"
+          },
+         "locale": {
+           "description": "locale tag source ",
+           "type": "string"
+         },
+         "options": {
+           "description": "Formatting paramters for the number",
+           "type": "object",
+           "properties": {
+             "granularity": {
+               "type": "string",
+               "description": "What kind of segmentation to do",
+               "enum": ["grapheme", "word", "sentence"]
```

Comment:
I prefer `grapheme_cluster` over `grapheme` because that is more precise. `grapheme` is confusing, even if it comes from ECMA.

Reply:
OK